src/lib/storage.js

```diff
@@ -46,7 +49,7 @@ class Storage extends ScratchStorage {
     }
     getProjectUpdateConfig (projectAsset) {
         return {
-            url: `${this.projectHost}/${projectAsset.assetId}`,
+            url: `${this.projectHost}/${projectAsset.assetId}?token=${this.projectToken}`,
```

Comment:
I don't think this is necessary, since the token will only be required on fetches.
```suggestion
            url: `${this.projectHost}/${projectAsset.assetId}`,
```